manifests/s/Samsung/PlayTouchLauncher/1.2.0.32/Samsung.PlayTouchLauncher.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Samsung.PlayTouchLauncher
+PackageVersion: 1.2.0.32

```

```suggestion
PackageVersion: 1.2.0
```
This should match control panel. The version folder will need to be renamed after committing this change.